<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Taschenrechner</class>
 <widget class="QMainWindow" name="Taschenrechner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>522</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>603</width>
    <height>522</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Taschenrechner</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="LabelEingabe1">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>180</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Eingabe1</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelEingabe2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>210</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Eingabe2</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelSumme">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>240</y>
      <width>50</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Summe</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Eingabe1">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>180</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="Eingabe2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>210</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QDialogButtonBox" name="buttonBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>290</y>
      <width>156</width>
      <height>23</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::Cancel</set>
    </property>
    <property name="centerButtons">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="LabelErgebnis">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>240</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>290</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Berechnen</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>603</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Taschenrechner</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>323</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
